packages/governance/src/question.js

```diff
@@ -90,23 +99,31 @@ const looksLikeIssueForType = (electionType, issue) => {
   switch (electionType) {
     case ElectionType.SURVEY:
     case ElectionType.ELECTION:
-      looksLikeSimpleIssue(/** @type {SimpleIssue} */ (issue));
+      assertSimpleIssue(/** @type {SimpleIssue} */ (issue));
       break;
     case ElectionType.PARAM_CHANGE:
-      looksLikeParamChangeIssue(/** @type {ParamChangeIssue} */ (issue));
+      assertParamChangeIssue(/** @type {ParamChangeIssue} */ (issue));
+      break;
+    case ElectionType.API_INVOCATION:
+      assertApiInvocation(/** @type {ApiInvocationIssue} */ (issue));
```

Comment:
remove casts. we don't know what they really are at this point

Reply:
done